BlackMarket/Views/Purchases/PurchasesListView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  PurchasesList.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 26/09/2023.
+//
+
+import SwiftUI
+
+struct PurchasesListView: View {
+  
+  @State var path: NavigationPath = .init()
+  
+  var body: some View {
+    NavigationStack(path: $path.animation(.easeOut)) {
+      VStack {
+        Text(LocalizedString.PurchasesList.purchases_list_hint)
+        Spacer().frame(height: 120)
+        Button(action: {
+          path.append(PendingReviewPurchasesView.tag)
+        }) {
+          Text(LocalizedString.PurchasesList.pending_review_purchases_button_text)
+        }
+        Spacer().frame(height: 10)
+        Button(action: {
+          path.append(CancelledPurchasesView.tag)
+        }) {
+          Text(LocalizedString.PurchasesList.cancelled_purchases_button_text)
+        }
+      }.navigationTitle(LocalizedString.PurchasesList.title)
+        .padding(.horizontal, 20)
+        .padding(.vertical, 15)
+        .navigationDestination(for: String.self) { tag in
```

Comment:
WDYT about managing all the destinations with an `enum type`, in that way we can manage just one Type of destination instead of doing one for `String` and another for `PurchaseDetail`

ie: 

```
enum Screens {
case PurchaseDetail
case PendingReview
case CanceledPurchases
...
}
```

you can explore [SUI Routing](https://github.com/rootstrap/SUI-Routing) library to get more context about that